ExtensionBundle.Tests/TestData/windows_extensions.deps.json

```diff
@@ -0,0 +1,5174 @@
+{
+  "runtimeTarget": {
+    "name": ".NETCoreApp,Version=v3.1",
+    "signature": ""
+  },
+  "compilationOptions": {},
```

Comment:
These files are taken from "any-any.zip" under published artifacts from main branch - [Link](https://azfunc.visualstudio.com/Azure%20Functions/_build/results?buildId=156488&view=artifacts&pathAsName=false&type=publishedArtifacts) for Windows and Linux.